checkr/src/models/candidate.ts

```diff
@@ -0,0 +1,44 @@
+import { Table, Column, Model, ForeignKey, BelongsTo, Length, IsEmail, isDataType } from 'sequelize-typescript';
+import { User } from './user';
+
+@Table
+export class Candidate extends Model<Candidate> {
+    @Column({ primaryKey: true, autoIncrement: true })
+    candidateId!: number;
+
+    @Length({ min: 3, max: 45 })
+    @Column
+    name!: string;
+
+    @IsEmail
+    @Length({ min: 6, max: 45 })
+    @Column
+    email!: string;
```

Comment:
search how to add index.

you can add index on email.

this will be helpful, when we filter candidates based on email.

add index on whichever column you, fill we will perform frequent search and filtering

Reply:
Yes, Need to explore this.

Reply:
Added index for few columns, I will add/remove during API implementation.